packages/unleash-frontend-next/src/component/feature/FeatureListContainer.jsx

```diff
@@ -0,0 +1,21 @@
+import { connect } from 'react-redux';
+import { toggleFeature } from '../../action';
+import FeatureList from './FeatureList';
+
+const mapStateToProps = (state) => ({
+    features: state.features,
+});
+
+const mapDispatchToProps = (dispatch) => ({
+    onFeatureClick: (id) => {
+        dispatch(toggleFeature(id));
```

Comment:
How? I had it like that first with just an object return, but then eslint complains about the [arrow-body-style](http://eslint.org/docs/rules/arrow-body-style).